lua/gruvbox-baby/theme.lua

```diff
@@ -381,6 +381,18 @@ function M.setup(config)
     })
   end
 
+  if config.hideInactiveStatusline then
+    local inactive = { style = "underline", bg = c.background_colored, fg = c.foreground, sp = c.foreground }
+
+    -- StatusLineNC
+    theme.base.StatusLineNC = inactive
+
+    -- LuaLine
+    for _, section in ipairs({ "a", "b", "c" }) do
+      theme.defer["lualine_" .. section .. "_inactive"] = inactive
```

Comment:
This will cause an error because `theme.defer` does not exists. you have to create a `theme.defer = {}` before the `if` condition to make it work.